Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -168,13 +159,91 @@ public struct SymbolRole: OptionSet {
   }
 }
 
+public enum IndexSymbolKindEnum: Int {
```

Comment:
I'm calling it `IndexSymbolKind` rather than `SymbolKind` because otherwise there are naming collisions in `SourceKit-LSP`. I cannot find a way to use library name spacing when there's also a class with the same name as the library.

Reply:
> I cannot find a way to use library name spacing when there's also a class with the same name as the library.

Not sure what you mean here: doesn't this only apply to the `IndexStoreDB` class itself (which has no conflict)?

Reply:
To differentiate this `SymbolKind` from the LanguageServerProtocol `SymbolKind`, I tried doing `IndexStoreDB.SymbolKind` (trying to specify that I want SymbolKind to come from the IndexStoreDB library), but this resulted in an error because it seemed to look for SymbolKind in the `IndexStoreDB` class rather than looking for it in the library.

I think my problem is similar to [this stackoverflow](https://stackoverflow.com/questions/26774101/swift-namespace-conflict)

Reply:
Got it.  `IndexSymbolKind` (without `Enum`) seems fine for now.